### PR TITLE
Promisified classes can't access their own context

### PR DESCRIPTION
**Pull request by Richienb**
Check the test.

cc @sindresorhus

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -269,6 +269,22 @@ test('class support - options.include over options.exclude', t => {
...
+	constructor() {
+		this.value = true;
+	}
+
+	getTrue(callback) {
+		callback(this.value);
+	}
+}
+
+test('class support - allows function to access their context', async t => {
```
Comment (sindresorhus) [inline test.js:282]:
```suggestion
test.failing('class support - allows function to access their context', async t => {
```

**Comment by sindresorhus** [comment]
As a workaround for now, you should be able to do `pify(trueReflector.getTrue.bind(trueReflector))`.

**Comment by sindresorhus** [comment]
// @frangio

**Comment by frangio** [comment]
The proposed test case is working as intended. `trueReflector.getTrue` is a free function that has no `this` attached, and `pify` can't work around that.

The right way to use the library here is to wrap the class instance and call the method on the wrapped object: `await pify(trueReflector).getTrue()`.

**Comment by sindresorhus** [comment]
@frangio Good point. I somehow thought there was a way to get the original class from the method and bind it. This is a common questions though (https://github.com/sindresorhus/pify/issues/79#issuecomment-626405280), so I think we should better document this.

@Richienb Would you be willing to add a `FAQ` section or something to the readme? If not, I'll open an issue about it.

**Comment by sindresorhus** [comment]
https://github.com/sindresorhus/pify/pull/80

**Pull request closed without merging**